blocklylib-vertical/src/main/java/com/google/blockly/android/ui/vertical/VerticalBlockViewFactory.java

```diff
@@ -133,6 +134,11 @@ protected FieldView buildFieldView(Field field) {
                 colorView.setWorkspaceHelper(mHelper);
                 break;
             }
+            case Field.TYPE_ANGLE: {
+                BasicFieldAngleView angleView = (BasicFieldAngleView) fieldView;
+                angleView.setWorkspaceHelper(mHelper);
+                break;
+            }
```

Comment:
Not sure why this block is needed.  I expect it would be captured in the default case below, where super.buildField() is also called.